android/src/com/frostwire/android/gui/adapters/OnFeedbackClickAdapter.java

```diff
@@ -70,8 +70,7 @@ public void onClick(Fragment owner, View v) {
                 Build.MODEL,
                 Build.DEVICE,
                 Build.BOARD,
-                Build.CPU_ABI,
-                Build.CPU_ABI2);
+                Build.SUPPORTED_ABIS);
```

Comment:
I have a feeling this might break the code.

These 2 values are used to format a string:
```java
String body = String.format("FrostWire for Android %s build %s %s \n\nAndroid SDK: %d\nAndroid RELEASE: %s (%s)\nManufacturer-Model: %s - %s\nDevice: %s\nBoard: %s\nCPU ABI: %s\nCPU ABI2: %s\n\n\n\n",
                Constants.FROSTWIRE_VERSION_STRING,
                Constants.FROSTWIRE_BUILD,
                Constants.IS_GOOGLE_PLAY_DISTRIBUTION ? "" : "(PLUS)",
                Build.VERSION.SDK_INT,
                Build.VERSION.RELEASE,
                Build.VERSION.CODENAME,
                Build.MANUFACTURER,
                Build.MODEL,
                Build.DEVICE,
                Build.BOARD,
                Build.CPU_ABI,
                Build.CPU_ABI2);
```

`Build.SUPPORTED_ABIS` can return an array of variable size, it's not guaranteed it will have 1 or 2 or more supported abis, the .format() call could break in runtime.

Reply:
can't accept this commit, but will merge the other 2 changes manually.

This is why it's good to separate the commits.